spring-boot-data-dynamodb/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>web - %date [%thread] %-5level %logger{36} - %message%n
+			</pattern>
+		</encoder>
+	</appender>
+
+	<root level="INFO">
+		<appender-ref ref="STDOUT" />
+	</root>
+
+</configuration>
```

Comment:
We have indentation issue here. Check out how it is done is other projects. Usually **4 spaces** preferred over **tab**. It's a minor issue but it is good to consistent with other projects.